images/linux/scripts/installers/android.sh

```diff
@@ -93,7 +94,13 @@ do
 done
 
 ANDROID_NDK_MAJOR_LATEST=(${ANDROID_NDK_MAJOR_VERSIONS[-1]})
+ndkDefaultFullVersion=$(get_full_ndk_version $ANDROID_NDK_MAJOR_DEFAULT)
 ndkLatestFullVersion=$(get_full_ndk_version $ANDROID_NDK_MAJOR_LATEST)
+ANDROID_NDK="$ANDROID_SDK_ROOT/ndk/$ndkDefaultFullVersion"
+# ANDROID_NDK, ANDROID_NDK_HOME, and ANDROID_NDK_LATEST_HOME variables should be set as many customer builds depend on them https://github.com/actions/virtual-environments/issues/5879
```

Comment:
Nit: didn't you mean:

```suggestion
# ANDROID_NDK, ANDROID_NDK_HOME, and ANDROID_NDK_ROOT variables should be set as many customer builds depend on them https://github.com/actions/virtual-environments/issues/5879
```
?

Reply:
@MarijnS95 ugh, you're right, thanks! Fixed here https://github.com/actions/virtual-environments/pull/6025

Reply:
It's just a comment, I wasn't expecting you to actually fix this :)